### PR TITLE
Wording improvements & prettierrc configuration

### PR DESCRIPTION
**Pull request by JasonTame**
## Context 
The goal of this PR is to improve the wording of the help instructions found in each step of the deployment process. 

### Wording
Please read through the wording changes to check if it flows nicely and is easy to understand

### Prettier
I noticed that my default prettier settings led to many formatting changes so I've added a simple `prettierrc` file with a couple of defaults that make sense in JS. Please check if you are happy with the config added there and if you'd recommend any other settings

### CODE REVIEW

File: src/App.vue [hunk 1]
```diff
@@ -52,7 +52,7 @@ export default {
     MilestoneSelector,
     Milestone,
     Release,
-    AfterDeployChecklist,
+    AfterDeployChecklist
```
Comment (antennaio) [inline src/App.vue:55]: The trailing commas are applied quite consistently, so you could enforce trailing commas in `.prettierrc` if you wanted to keep them.

https://prettier.io/docs/en/options.html#trailing-commas

**Review by antennaio — COMMENTED** [review]
I went through all the wording improvements and these are great and flow really well. 👌  @JasonTame 

Is `prettier` going to be used to apply standard formatting to files going forward? I'm asking, because there's a `yarn run format` command, but it makes use of `eslint` at the moment. Ideally, it would be nice to be able to run `yarn install` + `yarn run format` to get the desired formatting without having to worry about installing any additional dependencies, etc.

File: src/App.vue (hunk 1 above)
Comment (JasonTame) [reply src/App.vue]: Added 👍

**Comment by JasonTame** [comment]
@antennaio I decided to give my brain a short break from checkout and quickly resolve the issues here 😅  I found that the formatter wasn't actually working, so I fixed that and then added eslint rules to match prettier. I think we can use both, there are only a few rules and both eslint and prettier have their benefits.

File: package.json [hunk 2]
```diff
@@ -5,7 +5,7 @@
   "scripts": {
     "serve": "vue-cli-service serve",
     "build": "vue-cli-service build --modern",
-    "format": "yarn run eslint --fix --ext .js,.vue src,tests",
+    "format": "yarn run eslint --fix --ext .js,.vue src/**",
```
Comment (lptn) [inline package.json:8]: **question:** ‏why tests are excluded?

**Review by lptn — APPROVED** [review]
👍

File: package.json (hunk 2 above)
Comment (JasonTame) [reply package.json]: It wasn't working with the comma as a separator so I wasn't sure it was possible to specify both directories in one command, but I've found that it works fine if you separate them with a space, fix has been pushed 🙏

**Pull request merged**